react-electron-poc/src/renderer/components/layout/Layout.tsx

```diff
@@ -100,134 +242,212 @@ const Layout = () => {
+                    onClick={() => addTab()}
+                    onContextMenu={(e: React.MouseEvent<HTMLButtonElement>) => {
+                        addTab('Erb');
+                        e.preventDefault();
+                    }}
+                >
+                    Add
+                    <select
+                        className="layout-interactive embedded-input tab-type-select"
+                        onClick={(e: React.MouseEvent<HTMLSelectElement>) => {
+                            // Do not add a tab when we click the dropdown
+                            e.stopPropagation();
+                        }}
+                        value={newTabType}
+                        onChange={(e: React.ChangeEvent<HTMLSelectElement>) =>
+                            updateNewTabType(e.target.value as AddPanelType)
                         }
-                    />
+                    >
+                        {ADD_SCRIPTURE_PANEL_TYPES.map((addPanelType) => (
+                            <option value={addPanelType}>
+                                {addPanelType.replace('ScriptureTextPanel', '')}
+                            </option>
+                        ))}
+                    </select>
+                    Tab
+                </button>
+                <span className="settings">
+                    <label className="layout-interactive">
+                        Starting Tabs:
+                        <input
+                            type="number"
+                            className="layout-interactive embedded-input"
+                            value={startingTabs}
+                            onChange={(
+                                e: React.ChangeEvent<HTMLInputElement>,
+                            ) =>
+                                updateStartingTabs(parseInt(e.target.value, 10))
+                            }
+                        />
+                    </label>
+                    <label className="layout-interactive">
+                        Use Virtualization
+                        <input
+                            type="checkbox"
+                            checked={useVirtualization}
+                            onChange={(event) =>
+                                updateUseVirtualization(event.target.checked)
+                            }
+                        />
+                    </label>
+                    <label className="layout-interactive">
+                        Edit whole book
+                        <input
+                            type="checkbox"
+                            checked={browseBook}
+                            onChange={(event) =>
+                                updateBrowseBook(event.target.checked)
+                            }
+                        />
+                    </label>
                 </span>
             </div>
             <div className="layout-dock">
+                {/* <ScriptureTextPanelSlate
```

Comment:
Are you leaving this comment there as an option that you can come back to as needed?

Reply:
Yep! I think this one may be worth leaving in if that's ok - doing this eliminates some variables when trying to optimize slate :)